.github/workflows/smoke.yml

```diff
@@ -36,7 +35,7 @@ jobs:
       uses: actions/checkout@v3
 
     - name: Setup LXD
-      uses: ./.github/setup-lxd
+      uses: canonical/setup-lxd@90d76101915da56a42a562ba766b1a77019242fd
```

Comment:
I know this has already been merged. Specially, for this reason, this should target a released version, not a SHA.

Reply:
doesn't have any releases yet, maybe will add one when it proves stable enough.